examples/platform_services/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,7 @@
 
     <application android:name="io.flutter.app.FlutterApplication" android:label="@string/app_name" >
         <activity
-            android:name=".ExampleActivity"
+            android:name="com.example.flutter.ExampleActivity"
```

Comment:
No. The original manifest package was wrong, and I was unaware of the . prefix shorthand. Fixed.